File: Prog1part1.c
/*
Jackson Hoornaert
jackson.hoornaert@und.edu
Part 1 - Using user input create a mailing adress using 6 print statements
*/

#include <stdio.h>

int main(){
    char first[20] = "a";
    char last[20] = "a";
    char address[75] = "a";
    char city[20] = "a";
    char state[20] = "a";
    char zip[20] = "a";

    printf("First: ");
    scanf("%[^\n]%*c", first);
    printf("Last: ");
    scanf("%[^\n]%*c", last);
    printf("Address: ");
    scanf("%[^\n]%*c", address);
    printf("City: ");
    scanf("%[^\n]%*c", city);
    printf("State: ");
    scanf("%[^\n]%*c", state);
    printf("Zip: ");
    scanf("%[^\n]%*c", zip);
    printf("\n");

    printf("%s ", first);
    printf("%s\n", last);
    printf("%s\n", address);
    printf("%s, ", city);
    printf("%s  ", state);
    printf("%s", zip);
}

File: Prog1part1.java
/*
 * Jackson Hoornaert
 * jackson.hoornaert@und.edu
 * Part 1 - Using user input create a mailing adress using 6 print statements
 */
import java.util.Scanner;

 public class Prog1part1{
    public static void main(String[] args){
        Scanner keyboard = new Scanner(System.in);

        System.out.print("First: ");
        String first = keyboard.nextLine();
        System.out.print("Last: ");
        String last = keyboard.nextLine();
        System.out.print("Address: ");
        String address = keyboard.nextLine();
        System.out.print("City: ");
        String city = keyboard.nextLine();
        System.out.print("State: ");
        String state = keyboard.nextLine();
        System.out.print("Zip: ");
        String zip = keyboard.nextLine();
        System.out.println();

        keyboard.close();

        System.out.print(first + " ");
        System.out.println(last);
        System.out.println(address);
        System.out.print(city + ", ");
        System.out.print(state + "  ");
        System.out.print(zip);
    }
 }